appeals/api/.env.test

```diff
@@ -25,3 +25,4 @@ BO_BLOB_STORAGE_ACCOUNT=https://127.0.0.1:10000/
 # - - - - - - - - - Swagger - - - - - - -
 SWAGGER_JSON_DIR=./src/server/openapi.json
 SRV_HORIZON_URL=http://horizontest
+MOCK_HORIZON='false'
```

Comment:
shouldn't this be true?

Reply:
No, because test is for the testing it bypasses the horizon call either way. Want to avoid adding more mocks to the mix that might break things